packages/core/src/database/migrations/z1645570144-delete-notifiedBacker.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        if (process.env.NODE_ENV === 'test') {
+            return;
+        }
+        await queryInterface.dropTable('notifiedbacker');
+    },
+
+    down(queryInterface, Sequelize) {},
```

Comment:
Leave this on the "future migrations" folder (later on we can move to migrations folder). Remember that we have pre-load. If we do both these on the same push, it's going to crash the running instance.

Reply:
done!